micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/CacheMeterBinder.java

```diff
@@ -40,6 +40,8 @@ public abstract class CacheMeterBinder<C> implements MeterBinder {
 
     private static final String DESCRIPTION_CACHE_GETS = "The number of times cache lookup methods have returned a cached (hit) or uncached (newly loaded or null) value (miss).";
 
+    static final long UNSUPPORTED = -1L;
```

Comment:
I'm not happy with this magic number to express an unsupported meter for the `long` primitive type but couldn't find a better way without breaking changes.